AnyFileViewer/Sources/AnyFileViewer/Models/AnyFileModel.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  AnyFileModel.swift
+//  AnyFileViewer
+//
+//  Created by David Martin Nevado on 4/3/25.
+//
+
+public struct AnyFileModel {
+    public let url: String
+    public let ext: String
```

Comment:
can we change ext: String by an enum
```
public enum FileExtensionType: String {
        case png = "png"
        case pdf = "pdf"
        case mp4 = "mp4"
        // etc..
}
```

then having this we prevent the misswriting the extension type at the moment when we instantiate the model.

```suggestion
    public let ext: FileExtensionType
```

<br>

```
AnyFileModel(url: "https://pdfobject.com/pdf/sample.pdf", ext: .pdf)
```